src/components/AddMovie/index.js

```diff
@@ -1,5 +1,7 @@
 import React from "react";
-import { AddMovieButtonContainer, AddMovieButton } from "./indexStyles";
+
+//styles
+import { AddMovieButtonContainer, AddMovieButton } from "./styles";
 
 export const AddMovie = ({ addMovieModal }) => {
   return (
```

Comment:
Such "ADD MOVIE" text content should also be part of I18Y logic.

Reply:
logic has been added to the Add movie component